falco_event/src/events/types.rs

```diff
@@ -1780,7 +1780,7 @@ event_info! {
                                     EF_CREATES_FD | EF_MODIFIES_STATE,
                                     5,
                                     {{"dirfd", PT_FD, PF_DEC},
-                                     {"name", PT_FSRELPATH, PF_NA, 1},
+                                     {"name", PT_FSRELPATH, PF_NA, 0},
```

Comment:
?
```suggestion
                                     {"name", PT_FSRELPATH, PF_NA, DIRFD_PARAM(0)},
```

Reply:
oh no, i see

Reply:
Yeah, we do some mild preprocessing on the C file, I didn't really want to write a full blown C parser in Rust proc macros :upside_down_face: